proposals/simd/SIMD.md

```diff
@@ -285,14 +285,40 @@ The input lane value, `x`, is interpreted the same way as for the splat
 instructions. For the `i8` and `i16` lanes, the high bits of `x` are ignored.
 
 ### Shuffle lanes
+
+#### Immediate permutation rule
 * `v8x16.shuffle(a: v128, b: v128, imm: ImmLaneIdx32[16]) -> v128`
 
 Returns a new vector with lanes selected from the lanes of the two input vectors
-`a` and `b` specified in the 16 byte wide immediate mode operand `imm`. This
-instruction is encoded with 16 bytes providing the indices of the elements to
+`a` and `b` specified in the 12 byte wide immediate mode operand `imm`. This
```

Comment:
Why 12 bytes ?

Reply:
Each index in imm is in `[0, 31]`: meaning 5 bits.
So the complete immediate is `5 * 16 = 80` bits, which is 12 bytes.

Reply:
I must be really thick today, isn't that 10 bytes ? (or am I missing alignment requirements?)

Reply:
My bad ^^", you're right, it is actually 10 bytes.
My brain mixed it up with the 12 bits from v32x4...